src/GeneExpressionTSnePlot.js

```diff
@@ -123,18 +171,57 @@ const GeneExpressionScatterPlot = (props) => {
                        defaultSpecies={speciesName}
                        onSelect={ (event) => { onSelectGeneId(event) } }
     />,
-
+    //react-responsive design
+    <Desktop key={`Desktop`}>
```

Comment:
This block of code seems a bit repetitive. Can you think of any way to refactor the code? Also, please be careful with your code indentation (`ScatterPlotLoader` is a child of `Desktop` and the indentation should reflect that).